plcc/src/proteinstructure/Molecule.java

```diff
@@ -157,12 +151,90 @@ public String getAltLocWithMostAtoms() {
         return maxAltLoc;
     }
     
+    /**
+     * 
+     */
+    public ArrayList<Atom> chooseYourAltLoc() {
+        System.out.println("geht in super");
```

Comment:
Do we need this print? ;) I like to flag my debug prints with a comment DEBUG or TODELETE and search for them before commiting to avoid commiting them.